controllers/profiles.js

```diff
@@ -0,0 +1,129 @@
+const fs = require("fs")
+const profiles = require("../data/profiles.json")
+
+const errorResHelper = (errorDescription, res) => {
+  res.status(400).send(
```

Comment:
to keep the format of the responses, you'd better return a json instead of a text